pkcs11/util_pkcs11.h

```diff
@@ -62,7 +62,7 @@ bool check_verify_mechanism(yubihsm_pkcs11_slot *slot,
 CK_RV apply_verify_mechanism_init(yubihsm_pkcs11_op_info *op_info);
 CK_RV apply_verify_mechanism_update(yubihsm_pkcs11_op_info *op_info,
                                     CK_BYTE_PTR in, CK_ULONG in_len);
-CK_RV apply_verify_mechanism_finalize(yubihsm_pkcs11_op_info *op_info);
+CK_RV apply_verify_mechanism_finalize();
```

Comment:
We shouldn't drop this argument (I'm fine with discussing if the function is at all needed, but since we call it with an op_info we shouldn't declare the function like this).

Reply:
Fixed